tutorials/io/saving_games.rst

```diff
@@ -112,17 +112,17 @@ like this:
 
 
 This gives us a dictionary with the style
-``{ "variable_name":that_variables_value }`` which will be useful when
+``{ "variable_name":that_variable_s_value }``, which will be useful when
```

Comment:
I can see why you made this change, but the way it looks after your change seems weird to me. Maybe `that_variable_value` (remove the `s`)? Or maybe `variable_value` to match up with `variable_name`.

Reply:
> I can see why you made this change, but the way it looks after your change seems weird to me. Maybe `that_variable_value` (remove the `s`)? Or maybe `variable_value` to match up with `variable_name`.

@LikeLakers2 : Additional option: ``value_of_variable``.

``variable_value`` almost sounds like "arbitrary/un-deterministic/stochastic/random value" to me, while ``that_variable_value`` merely qualifies which one you mean.